llvm/lib/Object/DXContainer.cpp

```diff
@@ -273,9 +272,9 @@ Error DirectX::RootSignature::parse(StringRef Data) {
   Flags = support::endian::read<uint32_t, llvm::endianness::little>(Current);
   Current += sizeof(uint32_t);
 
-  assert(Current == Begin + RootParametersOffset);
+  assert(Current == PartData.begin() + RootParametersOffset);
```

Comment:
I don't think this assert should be here.  The code isn't making this assumption, and the binary format doesn't require it either.